src/js/post-911-gib-status/actions/post-911-gib-status.js

```diff
@@ -50,3 +52,29 @@ export function getEnrollmentData() {
       });
   };
 }
+
+export function getServiceUp() {
+  return (dispatch) => {
+    dispatch({
+      type: SET_SERVICE_UP,
+      serviceUp: SERVICE_UP_STATES.pending
+    });
+
+    return apiRequest(
+      '/backend_statuses/gibs'
+    ).then((response) => {
+      const availability = response.data.attributes.isAvailable;
+
+      dispatch({
+        type: SET_SERVICE_UP,
+        serviceUp: availability ? SERVICE_UP_STATES.up : SERVICE_UP_STATES.down
```

Comment:
The API also returns the `name` (`response.data.attributes.name`) if you care to dispatch that attribute as well.  It represents the name of the service that status was just fetched for.

![image](https://user-images.githubusercontent.com/3077884/37625656-3348e2ca-2ba3-11e8-93e3-815f1a072dd7.png)

Granted, `gibs` is the only one that does anything intelligent :-P for now.

Reply:
If we call `/v0/backend_statuses/gibs`, it'll only return the status for gibs, right?

Reply:
Correct.